templates/vue/cypress-sample.json

```diff
@@ -5,7 +5,7 @@
     "SUBSCRIBER_USERNAME": "subscriber_name",
     "SUBSCRIBER_PASSWORD": "subscriber_password"
   },
-  "baseUrl": "http://localhost:8000",
```

Comment:
Is this change necessary?

Reply:
It is for ease of setting up, because for me I had to change the port here to 8888 when copying this sample to my own config. Since the wordpress site has a default port of 8888, I think changing it in the cypress-sample.json would be appropriate.